db/migrate/20230626112559_add_team_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddTeamToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :team, :string
```

Comment:
I should have raised this earlier - sorry!

We can populate the `user.team` for some users and should do so, I wrote it for `project.team` but missed it for this work! 😞 

- When `user.caseworker?` is true, `user.team == "regional_casework_services"`
- When `user.team_leader?` is true `user.team == "regional_casework_services"`
- When `user.service_support?` is true `user.team == "service_support"`

We can do this once this gets deployed if you prefer that option?

Reply:
I can add it here! I actually forgot to add it to the Projects one as well 😆

Reply:
Projects one is on another card so you can leave it: https://trello.com/c/NB8mmAFA